packages/hd/tests/keystore.test.ts

```diff
@@ -52,24 +55,56 @@ test("load and check password, loads private key", (t) => {
  * MAC: 2103ac29920d71da29f15d75b4a16dbe95cfd7ff8faea1056c33131d846e3097
  * Cipher key: fac192ceb5fd772906bea3e118a69e8b
  */
+// This case does NOT work with @crpyto/hashes/scrypt because N is not less than 2^(128 * r / 8)
+// const json = {
+//   crypto: {
+//     cipher: "aes-128-ctr",
+//     cipherparams: {
+//       iv: "83dbcc02d8ccb40e466191a123791e0e",
+//     },
+//     ciphertext:
+//       "d172bf743a674da9cdad04534d56926ef8358534d458fffccd4e6ad2fbde479c",
+//     kdf: "scrypt",
+//     kdfparams: {
+//       dklen: 32,
+//       n: 262144, // 2^18 Scrypt: N must be larger than 1, a power of 2, less than 2^(128 * r / 8) and less than 2^32
+//       r: 1,
+//       p: 8,
+//       salt: "ab0c7876052600dd703518d6fc3fe8984592145b591fc8fb5c6d43190334ba19",
+//     },
+//     mac: "2103ac29920d71da29f15d75b4a16dbe95cfd7ff8faea1056c33131d846e3097",
+//   },
+//   id: "3198bc9c-6672-5ab3-d995-4942343ae5b6",
+//   version: 3,
+// };
 test("load test vector keystore", (t) => {
+  // const privateKey = "0xe8f32e723decf4051aefac8e2c93c9c5b214313817cdb01a1494b917c8436b35";
+  // const chainCode = "0x873dff81c02f525623fd1fe5167eac3a55a049de3d314bb42ee227ffed37d508";
+  // const keystore = KeyStore.create(
+  //   new ExtendedPrivateKey(privateKey, chainCode),
+  //   'testpassword',
+  //   {
+  //     iv: bytify("0x8f7097ee22add66106055bf3e241fece"),
+  //     salt: bytify("0x481ead5b0311079d2adcd74f32e3db3bdc5f447bc2217ad8164d2d7ead19079f")
+  //   }
+  // );
   const json = {
```

Comment:
Please

- comment out the entire test case without modifying it
- add a new case and paste the permanent URL of the test vector here

Reply:
OK. Done.